sdks/python/apache_beam/runners/interactive/interactive_runner_test.py

```diff
@@ -0,0 +1,104 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Tests for google3.pipeline.dataflow.python.interactive.interactive_runner.
+
+This module is experimental. No backwards-compatibility guarantees.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+
+import apache_beam as beam
+from apache_beam.runners.direct import direct_runner
+from apache_beam.runners.interactive import interactive_runner
+
+
+def print_with_message(msg):
+
+  def printer(elem):
+    print(msg, elem)
+    return elem
+
+  return printer
+
+
+class InteractiveRunnerTest(unittest.TestCase):
+
+  def test_basic(self):
+    # TODO(qinyeli) remove explicitly overriding underlying runner
+    # once interactive_runner works with FnAPI mode
```

Comment:
Could you clarify what the issue is? This will make tests pass, but be broken for users.